app/network/entity/entity.form.component.js

```diff
@@ -229,5 +233,72 @@
     function storeGroups(groups) {
       _.setContents(entityForm.groups.items, groups);
     }
+
+    function incrementBulk() {
+      var is_bulk = entityForm.v4.is_bulk;
+      var increment_vlan = entityForm.v4.increment_vlan;
+      var ip_increment = entityForm.input.v4.increment_amount;
+
+      if (is_bulk) {
```

Comment:
when I get these long if statements, I prefer to early exit the function with return, that way we don't have all this code double-tabbed in. Usually, it's ideal not to have more than 2 nested levels in a function - at that point, some of the logic should probably be broken off into a separate function or refactored to use early returns when possible.

```suggestion
      if (!is_bulk || !ip_increment) {
        return;
      }
      
      // remaining code.
```